### PR TITLE
Finish more tests

### PR DESCRIPTION
**Pull request by srallen**
The WIP part is I'm still figuring out how to do integration tests, but this doesn't need to wait on that. I did a lot of folder organization, clean up, and minor fixes for other tests. If all looks good, it can go ahead and squash and merge.

### CODE REVIEW

**Comment by wgranger** [comment]
I'm unable to get webpack to compile to open this up for viewing.

File: test/modules/organizations/containers/organizations-list-container-test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <OrganizationsListContainer />
+      </Provider>,
+    );
+  });
+
+  // describe('componentDidMount', function() {
+  //   it('calls fetchOrganizations once', function() {
+  //     expect(fetchOrganizationsSpy.calledOnce).to.be.true;
+  //   });
+  // });
```
Comment (wgranger) [inline test/modules/organizations/containers/organizations-list-container-test.js]: Could you add a comment as to why these are commented out, as opposed to being removed.

Comment (srallen) [inline test/modules/organizations/containers/organizations-list-container-test.js]: I need to figure out the integration testing both with the API and internal to the component. This is related to the above commented lines where I'm trying out rewire.

Comment (srallen) [inline test/modules/organizations/containers/organizations-list-container-test.js]: I can definitely add a comment inline about that.

**Comment by srallen** [comment]
Ok, it should be compiling now.

**Comment by wgranger** [comment]
This branch is compiling now. Looks like there was a style accidentally added to the layout though that is causing the layout below: 

screen shot 2017-05-17 at 3 46 39 pm

**Comment by srallen** [comment]
Bad rebase is fixed. Sorry about that. This should just be the tests I added (also now squashed)

**Comment by wgranger** [comment]
Merge conflict here, then I'll give it another go and hopefully it should be up to speed.

**Comment by srallen** [comment]
Ok it's resolved

**Comment by wgranger** [comment]
LGTM.

**Pull request merged**